install/migrations/update_10.0.9_to_10.0.10/ldap_fields.php

```diff
@@ -64,7 +64,23 @@
 /** /Fix non encoded LDAP fields in groups */
 
 /** Fix non encoded LDAP fields in users */
-$users = getAllDataFromTable('glpi_users', ['authtype' => 3]);
+$users = $DB->request([
+    'SELECT' => [
+        'glpi_users.id',
+        'glpi_users.user_dn',
+        'glpi_users.sync_field',
+    ],
+    'FROM'   => 'glpi_users',
+    'WHERE'  => [
+        'authtype' => 3,
+        [
+            'OR' => [
+                'user_dn' => ['REGEXP' => '(<|>|(&(?!#?[a-z0-9]+;)))'],
+                'sync_field' => ['REGEXP' => '(<|>|(&(?!#?[a-z0-9]+;)))'],
```

Comment:
```suggestion
                'user_dn' => ['REGEXP', '(<|>|(&(?!#?[a-z0-9]+;)))'],
                'sync_field' => ['REGEXP', '(<|>|(&(?!#?[a-z0-9]+;)))'],
```

Reply:
Since regexp is now in the SQL query, is should be useless in PHP code.